src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -224,7 +226,15 @@ class ThumbnailsPanel extends React.PureComponent {
   };
 
   onDocumentLoaded = () => {
-    const { setSelectedPageThumbnails } = this.props;
+    const { setSelectedPageThumbnails, dispatch } = this.props;
+
+    let doc = core.getDocument();
+    if (doc.type !== workerTypes.PDF) {
```

Comment:
If a user loads an office file, we will disable thumbnail operations, which makes sense to me. But if the user then loads a PDF document, do we want to re-enable the controls back?

Reply:
I changed it so if the controls are enabled and the user open a xod file the controls disappear. If they open a pdf after, as long as they didn't use "disableFeature" or something, the controls will be re-enabled. 

One thing to note is I added a "showShouldControls" flag on the state of the thumbnailsPanel instead of the global store. I could move it to the global store but I'm not sure how useful it would be at this point